services/src/main/java/io/druid/cli/validate/DruidJsonValidator.java

```diff
@@ -73,7 +86,23 @@ public void run()
       System.out.printf("File[%s] does not exist.%n", file);
     }
 
-    final ObjectMapper jsonMapper = new DefaultObjectMapper();
+    final Injector injector = GuiceInjectors.makeStartupInjector();
```

Comment:
@drcrallen Thanks for the code but it's almost the same I've did, which made exception `Exception in thread "main" java.lang.IllegalArgumentException: Unrecognized inject value id type (com.google.inject.Key), expecting String`. But it could be environment problem and if it works on yours, we can take this.